moveit_planners/ompl/ompl_interface/src/planning_context_manager.cpp

```diff
@@ -89,17 +89,23 @@ struct PlanningContextManager::CachedContexts
   std::mutex lock_;
 };
 
-MultiQueryPlannerAllocator::~MultiQueryPlannerAllocator()
+bool MultiQueryPlannerAllocator::storePlannerData()
 {
   // Store all planner data
   for (const auto& entry : planner_data_storage_paths_)
   {
     ob::PlannerData data(planners_[entry.first]->getSpaceInformation());
     planners_[entry.first]->getPlannerData(data);
+    if (data.numVertices() == 0)
```

Comment:
Please add an debug info for this here. The error in `OMPLInterface::storePlannerData()` shouldn't make assumptions about the reason for failures in this function.

Reply:
Added debug info, please check if this is what you meant.